src/renderers/webgl/WebGLTextures.js

```diff
@@ -274,8 +274,6 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 		const renderTargetProperties = properties.get( renderTarget );
 		const textureProperties = properties.get( texture );
 
-		if ( ! renderTarget ) return;
```

Comment:
This line is part of `deallocateTexture()` since `WebGLTextures` was created. I don't think the `if` statements ever evaluates to `true` though since a render target should always be set when `WebGLRenderTarget.dispose()` is executed.